### DISCUSSION TITLE
Donation or sponsor link?

### DISCUSSION THREAD

**Discussion by albertkun** [godotengine/godot-vscode-plugin#614] [Q&A] [answered]
Hello! The latest version of this plug-in (on Godot 3.5) is amazing and I'd like to see if I can donate to it! Otherwise I'll just donate to Godot Engine as per the current sponsor links! Thank you!

**Comment by DaelonSuzuka** [comment, marked as answer]
Hey, thank you so much! It's awesome to see people enjoying the extension.

If you want to donate specifically to the development of this extension, the best way to do that is probably my ko-fi link, which you can find on my GitHub profile under my avatar.

**Reply by albertkun** [reply]
Thank you for the reply! Sure, I'll go ahead and do that! Seriously, thank you so much for this plugin!
